backend/app/api/endpoints/runtime_work.py

```
@@ -630,22 +630,31 @@ def _resolve_runtime_work_backend_url(fastapi_request: Request) -> str:
     return f"{backend_url.rstrip('/')}{settings.API_PREFIX}/runtime-work"
 
 
-@router.post("/llm-responses-proxy/{token}/responses")
+@router.post("/llm-responses-proxy/responses")
 async def llm_responses_proxy_endpoint(
-    token: str,
     fastapi_request: Request,
     db: Session = Depends(get_db),
```

Ruff (0.15.20)

[warning] Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)